addons/sourcemod/scripting/AFKManager.sp

```diff
@@ -197,6 +198,7 @@ void InitializePlayer(int client)
 		ResetPlayer(client);
 		g_Players_iLastAction[client] = GetTime();
 		g_Players_bEnabled[client] = true;
+		CreateTimer(g_fKickTime, Timer_CheckPlayerHasJoinTeam, GetClientUserId(client), TIMER_FLAG_NO_MAPCHANGE);
```

Comment:
why do you create a timer to check this, cant we just compare it directly in the Timer_CheckPlayer function ?

Reply:
Team_none is only assigned when player didnt select any team on connect, running a check every 5 seconds (Timer_CheckPlayer) add extra check for something who can't be triggered after player selected a team.
I will go with this solution https://github.com/srcdslab/sm-plugin-AFKManager/pull/15#discussion_r1436095576